contracts/ScheduleTestHarness.sol

```diff
@@ -8,8 +8,9 @@ import {ScheduleUtils, Schedule} from "./ScheduleUtils.sol";
  *
  * Not intended for deployment in productionish environments.
  */
-
 contract ScheduleTestHarness {
+  // todo consider TestHarness vs Mock naming convention
```

Comment:
They are sort of different though since one is require to use a library and the other is a layer of a contract.. might leave them both but move them into a mock or test parent folder